core/deployment/src/main/java/io/quarkus/deployment/pkg/ManifestConfig.java

```diff
@@ -14,6 +14,15 @@
     @ConfigItem(defaultValue = "true")
     public boolean addImplementationEntries;
 
+    /**
+     * Custom manifest attributes to be added to the main section of the MANIFEST.MF file.
+     * An example of the user defined property:
+     * quarkus.package.manifest.attributes.{Entry-key1}={Value1}
+     * quarkus.package.manifest.attributes.{Entry-key2}={Value2}
```

Comment:
```suggestion
     * <pre>
     * quarkus.package.manifest.attributes."Entry-key1"=value1
     * quarkus.package.manifest.attributes."Entry-key2"=value2
     * </pre>
```
Having the curly braces can be confusing as we might think they are required for things to work.

Reply:
I removed curly braces from the manifest sections property comment too.